pom.xml

```diff
@@ -74,6 +74,7 @@
         <hamcrest.version>1.3</hamcrest.version>
         <randomizedtesting.version>2.1.16</randomizedtesting.version>
         <groovy.version>2.4.4</groovy.version>
+        <mustache.version>0.8.18</mustache.version>
```

Comment:
Better use the [same version as Elasticsearch 2.0](https://github.com/elastic/elasticsearch/blob/2.0/pom.xml#L308) which is `0.8.13`.